### ISSUE TITLE
Copy table & column doesn't preserve default column values

### ISSUE THREAD

**Issue by elring** [dbeaver/dbeaver#11989]
OS: Linux Mint 20
Dbeaver: 21.0.2
DB: PostgreSQL 12

After table copy (for example Ctrl+C and Ctrl+V in table list) new table doesn't have column default values.

Original table:
```
CREATE TABLE public.testtable (
	id serial NOT NULL,
	testchar varchar NULL DEFAULT 'aaaaa'::character varying,
	CONSTRAINT testtable_pkey PRIMARY KEY (id)
);
```
After Ctrl+C and Ctrl+V the table has no default values and DDL looks like:
```
CREATE TABLE public.testtable_1 (
	id serial NOT NULL,
	testchar varchar NULL,
	CONSTRAINT testtable_pkey_1 PRIMARY KEY (id)
);
```

**Comment by uslss** [comment]
thanks for the report

**Comment by LonwoLonwo** [comment]
Fixed in DBeaver 21.0.4

**Comment by uslss** [comment]
verified
